mmv1/products/blockchain/BlockchainNodes.yaml

```diff
@@ -0,0 +1,209 @@
+    description: |
+      ID of the requesting object.
+properties:
+  - !ruby/object:Api::Type::String
+    name: name
+    output: true
+    description: |
+      The number of the google project that firebase is enabled on.
+  - !ruby/object:Api::Type::String
+    name: createTime
+    output: true
+    description: |
+      The timestamp at which the blockchain node was first created.
+  - !ruby/object:Api::Type::String
+    name: updateTime
+    output: true
+    description: |
+      The timestamp at which the blockchain node was last updated.
+  - !ruby/object:Api::Type::KeyValueLabels
+    name: labels
+    description: |
+      User-provided key-value pairs
+  - !ruby/object:Api::Type::NestedObject
+    name: connectionInfo
+    output: true
+    description: |
+      The connection information through which to interact with a blockchain node.
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: endpointInfo
+        output: true
+        description: |
+          The endpoint information through which to interact with a blockchain node.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: jsonRpcApiEndpoint
+            output: true
+            description: |
+              The assigned URL for the node JSON-RPC API endpoint.
+          - !ruby/object:Api::Type::String
+            name: websocketsApiEndpoint
+            output: true
+            description: |
+              The assigned URL for the node WebSockets API endpoint.
+  - !ruby/object:Api::Type::NestedObject
+    name: ethereumDetails
+    description: |
+      User-provided key-value pairs
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: gethDetails
+        description: |
+          User-provided key-value pairs
+        properties:
+          - !ruby/object:Api::Type::Enum
+            name: garbageCollectionMode
+            immutable: true
+            values:
+              - :GARBAGE_COLLECTION_MODE_UNSPECIFIED
+              - :FULL
+              - :ARCHIVE
+            description: |
+              Blockchain garbage collection mode.
+      - !ruby/object:Api::Type::NestedObject
+        name: additionalEndpoints
+        description: |
+          User-provided key-value pairs
+        properties:
+          - !ruby/object:Api::Type::String
+            name: beaconApiEndpoint
+            output: true
+            description: |
+              The assigned URL for the node's Beacon API endpoint.
+          - !ruby/object:Api::Type::String
+            name: beaconPrometheusMetricsApiEndpoint
+            output: true
+            description: |
+              The assigned URL for the node's Beacon Prometheus metrics endpoint.
+          - !ruby/object:Api::Type::String
+            name: executionClientPrometheusMetricsApiEndpoint
+            output: true
+            description: |
+              The assigned URL for the node's execution client's Prometheus metrics endpoint.
```

Comment:
I think you need to mark the additionalEndpoints field as output: true, in addition to what you've already done for the properties inside. That's the only reason I can think of right now for the latest acceptance test failure due to a non-empty plan